tests/providers/google/cloud/operators/test_gcs_to_bigquery.py

```diff
@@ -29,6 +32,15 @@
 TEST_SOURCE_OBJECTS = ['test/objects/*']
 
 
+@pytest.mark.backend("mysql", "postgres")
+@pytest.mark.credential_file(GCP_GCS_KEY)
+class TestGoogleCloudStorageToBigQueryExample(GoogleSystemTest):
+
+    @provide_gcp_context(GCP_GCS_KEY)
```

Comment:
```suggestion
    @provide_gcp_context(GCP_BIGQUERY_KEY)
```
 An exact description of the keys is not publicly available, but GCP_GCS_KEY only allows access to GCS. GCP_BIGQUERY_KEY allows access to BigQuery and GCS.

Reply:
I was not aware of that. Was developing a "project admin" service account. thanks